dev-tools/mage/pkg.go

```diff
@@ -348,9 +347,7 @@ func TestPackages(options ...TestPackagesOption) error {
 	args = append(args, "-files", MustExpand("{{.PWD}}/build/distributions/*"))
 
 	if out, err := goTest(args...); err != nil {
-		if !mg.Verbose() {
-			fmt.Println(out)
-		}
+		fmt.Println(out)
```

Comment:
Do we want to print it even in non-verbose mode?

Reply:
:-) The condition was to print when we weren't in Verbose.  But we always want to print the stdout, if we don't all we get is "Error 1", with no idea which of these tests failed.  That is because this is totally separate `go test` process.